src/sdk/namespace_cache.js

```diff
@@ -759,6 +818,10 @@ class NamespaceCache {
         return hub_res.value;
     }
 
+    function update_cache_stats_hook(bucket_name) {
+        return write_bytes => this.stats_collector.update_cache_stats({ bucket_name,write_bytes })
```

Comment:
you need to use semicolons to end statements.
You can run `npm run lint` locally to make sure before pushing..

Reply:
Oh I didn't know about that command. That sounds good